gdquest_art_tools/Manual.md

```diff
@@ -86,3 +86,10 @@ erase the text you searched for.
 The rename tool is smarter with meta tags. Writing `e=` will remove the
 extension tag entirely. For example, `Godete e=png s=50,100` will become
 `Godette s=50,100`.
+
+## COA Tools format
```

Comment:
So this part should be clarified on how it works, with each group layer being a coa_tools document of sorts. Thanks!

Reply:
Yeah, tried to fix it in 0173641